ui_framework/doc_site/src/views/tabs/tabs_example.js

```diff
@@ -8,30 +9,30 @@ import {
   GuideText,
 } from '../../components';
 
-const html = require('./tabs.html');
-const js = require('raw!./tabs.js');
+import Tabs from './tabs';
+const tabsSource = require('!!raw!./tabs');
+const tabsHtml = renderToHtml(Tabs);
 
 export default props => (
   <GuidePage title={props.route.name}>
     <GuideSection
       title="Tabs"
       source={[{
-        type: GuideSectionTypes.HTML,
-        code: html,
-      }, {
         type: GuideSectionTypes.JS,
-        code: js,
+        code: tabsSource,
+      }, {
+        type: GuideSectionTypes.HTML,
+        code: tabsHtml,
       }]}
     >
       <GuideText>
-        Wrap any series of components, e.g. Panel, in the VerticalRhythm component to space
-        them apart.
+        The children of the KuiTabs can be anything that can be rendered: they will be wrapped into KuiTab components.
```

Comment:
Can we tweak this slightly?

```jsx
The <GuideCode>KuiTabs</GuideCode> component accepts nodes as children. Each child will be wrapped within a <GuideCode>KuiTab</GuideCode> component.
```

You'll just need to import `GuideCode` from `../../components` on line 10.